webpack.config.js

```diff
@@ -55,7 +55,7 @@ const commonEntryFiles = {
 };
 
 const electronEntryFiles = {
-    deviceConnectView: [path.resolve(__dirname, 'src/electron/views/device-connect-view/device-connect-view-initializer.ts')],
+    deviceConnectView: [path.resolve(__dirname, 'src/electron/views/renderer-initializer.ts')],
```

Comment:
should we change `deviceConnectView` here? Doing this will requiere to update the `script` tag under `index.html` file

Reply:
Done; it's now called renderer and the index.html file is updated